mqm_tool.py

```diff
@@ -0,0 +1,391 @@
+import json
+import numpy as np
+import sys
+import os
+import csv
+from kd_tree import kdTree
+from utility import Utility
+from geoProcess import GeoProcessor
+import argparse
+import ast
+
+
+def road_count(in_road, in_grids, in_counts, out_folder, initial_bb):
+    """ Road count function.
+
+    This function calculates the number of flagged features in all of the grids for a given road file.
+    
+    Args:
+        in_road: a road file.
+        in_grids: a collection of bounding boxes.
+        in_counts: a count list in which every entry stores the number of flagged features to each grid.
+        out_folder: a folder path.
+        initial_bb: an initial bounding box.
+        
+    """
+    road_data = []
+    road_counts_his = np.zeros(len(in_grids))
+    csv_matrix = []
+    csv_matrix.append(['grid_id', 'err_roads', 'road_counts'])
```

Comment:
This can be combined with the line above.

Reply:
fixed.